main.js

```diff
@@ -1,31 +1,51 @@
+const { app, BrowserWindow } = require('electron');
 const path = require('path');
-const { app, BrowserWindow, ipcMain } = require('electron');
-
-const createWindow = () => {
-  const win = new BrowserWindow({
-    width: 800,
-    height: 600,
-    webPreferences: {
-        preload: path.join(__dirname, 'preload.js'),
-    },
-  })
-
-  ipcMain.handle('ping', () => 'pong')
-  win.loadFile('index.html')
+const serve = require('electron-serve');
+const loadURL = serve({ directory: 'public' });
+
+let mainWindow;
+
+function isDev() {
+    return !app.isPackaged;
 }
 
-app.whenReady().then(() => {
-  createWindow()
+function createWindow() {    
+    mainWindow = new BrowserWindow({
+        width: 800,
+        height: 600,
+        webPreferences: {
+            nodeIntegration: true,
+            preload: path.join(__dirname, 'preload.js')
+        },
+        icon: path.join(__dirname, 'public/favicon.png'),
+        show: false
+    });
 
-  app.on('activate', () => {
-    if (BrowserWindow.getAllWindows().length === 0) {
-      createWindow()
+    if (isDev()) {
+        mainWindow.loadURL('http://localhost:8080/');
+    } else {
+        loadURL(mainWindow);
     }
-  })
-})
-
-app.on('window-all-closed', () => {
-  if (process.platform !== 'darwin') {
-    app.quit()
-  }
-})
\ No newline at end of file
+    
+    // Open the DevTools and also disable Electron Security Warning.
+    // process.env['ELECTRON_DISABLE_SECURITY_WARNINGS'] = true;
+    // mainWindow.webContents.openDevTools();
```

Comment:
Are these comments left over from testing?

Reply:
Left it as a reminder to set this up properly during `dev`. I can gut it and track it properly on Jira?

Reply:
Naw I'm not too worried if it's a helpful comment 👍